src/providers/ad/ad_pac_common.c

```diff
@@ -24,9 +24,186 @@
 #include "providers/ad/ad_pac.h"
 #include "util/util.h"
 
+static bool
+compare_sid_with_dom_sid_and_rid(const struct dom_sid *sid,
```

Comment:
Wrap in `#ifdef HAVE_STRUCT_PAC_UPN_DNS_INFO_EX` to suppress compilation warning ('defined but not used') on platform missing modern Samba libs?

Reply:
fixed